ui/test-global-setup.ts

```diff
@@ -0,0 +1,3 @@
+export const setup = (): void => {
+  process.env.TZ = 'UTC'
```

Comment:
I would really like if we also checked how formatting works in other timezones, since all our internal interfaces use UTC+3

Reply:
I did this mainly for CI, otherwise the tests would give different results locally and on the CI machine, causing them to fail consistently. I believe the formatting works correctly since it’s the browser's responsibility to handle the timezone.